easybuild/easyconfigs/g/GROMACS/GROMACS-2024.1-foss-2023b.eb

```diff
@@ -74,12 +74,12 @@ exts_default_options = {
 }
 
 exts_list = [
-    ('gmxapi', '0.4.2', {
+    ('gmxapi', '0.5.0', {
         'preinstallopts': 'export CMAKE_ARGS="-Dgmxapi_ROOT=%(installdir)s ' +
                           '-C %(installdir)s/share/cmake/gromacs_mpi/gromacs-hints_mpi.cmake" && ',
-        'source_tmpl': 'gromacs-2023.3.tar.gz',
+        'source_tmpl': 'gromacs-2024.1.tar.gz',
```

Comment:
@bedroge @ocaisa We should use `%(version)s` here to avoid running into this again in the future?

And a comment on how to determine the `gmxapi` version that is shipped with GROMACS